sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -229,6 +273,29 @@ class AstBuilder extends DataTypeAstBuilder
     )
   }
 
+  override def visitDeclareCondition(ctx: DeclareConditionContext): ErrorCondition = {
+    val conditionName = ctx.multipartIdentifier().getText
+    val conditionValue = Option(ctx.stringLit()).map(_.getText).getOrElse("'45000'").
```

Comment:
why not just `.getOrElse("45000")` instead of `.getOrElse("'45000'").replace("'", "")

Reply:
We should replace ' also in the first case not only in `else` so I think this is good place to add call to `replace`. But either way I can do `.getOrElse("45000").replace("'", "")`

Reply:
ok, wasn't aware of it. then probably the cleanest way is `.map(_.getText.replace(...)).getOrElse("45000")`

Reply:
Done.